dev-db/redis/redis-3.2.9.recipe

```diff
@@ -0,0 +1,52 @@
+SUMMARY="Advanced key-value store"
+DESCRIPTION="
+Redis is an open source, advanced key-value store. It is often referred to as \
+a data structure server since keys can contain strings, hashes, lists, sets \
+and sorted sets.
+"
```

Comment:
I'll move those over tomorrow (if I get to it), copied these lines over from the previous version ...